src/itest/resources/features/company-insolvency.feature

```diff
@@ -6,12 +6,39 @@ Feature: Process company insolvency information
     When I send PUT request with payload "<data>" file
     Then I should receive 200 status code
     And the expected result should match "<result>" file
+    And the CHS Kafka API is invoked successfully
```

Comment:
Minor rearrangement: Can you please move all these scenarios into insolvency-error-retry.feature? that way we can have 3 features file in each component (happy path, error-retry and delete)